SEED-platform/seed
# 2047 taxlot dedicated map

#### Any background context you want to provide?
san-jose-test-taxlot.csv as test file for validation.

#### What's this PR do?
Added taxlot map for dedicated taxlot files.

#### How should this be manually tested?
Import san-jose-test-taxlot.csv and map taxlots properties. Expect to see a dedicated map for taxlot under inventory.

#### What are the relevant tickets?
#2047 Provide dedicated Tax Lot Map page

#### Screenshots (if appropriate)
Import 6 taxlots:
<img width="993" alt="Screen Shot 2020-07-24 at 7 41 56 AM" src="https://user-images.githubusercontent.com/49968203/88397629-7542aa80-cd81-11ea-95d9-0fdc2c6dff35.png">

Map shows 6 taxlots:
<img width="1234" alt="Screen Shot 2020-07-24 at 7 43 01 AM" src="https://user-images.githubusercontent.com/49968203/88397616-71168d00-cd81-11ea-8886-ecdc099edf08.png">

Added centroid for taxlots and property and taxlot taps for map:
<img width="809" alt="Screen Shot 2020-08-06 at 3 52 17 PM" src="https://user-images.githubusercontent.com/49968203/89586433-ef9d1100-d7fc-11ea-91b5-5762019c6f85.png">
